Library/Homebrew/keg_relocate.rb

```diff
@@ -16,9 +16,10 @@ def fix_dynamic_linkage
       link = file.readlink
       # Don't fix relative symlinks
       next unless link.absolute?
-      if link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
-        FileUtils.ln_sf(link.relative_path_from(file.parent), file)
-      end
+      next unless link.to_s.start_with?(HOMEBREW_CELLAR.to_s) || link.to_s.start_with?(HOMEBREW_PREFIX.to_s)
```

Comment:
> My boolean logic or my imagination might be failing me.

No but mine was which is why I had `unless ||` 😁 

Yeh, `next if !a && !b` and perhaps putting them into variables for length is a little easier to follow.